src/components/FeedbackForm.jsx

```diff
@@ -0,0 +1,140 @@
+import axios from 'axios';
+import { useState } from 'react';
+
+export default function FeedbackForm() {
+  const [submitted, setSubmitted] = useState(false);
+  const [valid, setValidity] = useState(false);
+  const [formValues, setFormValues] = useState({
+    firstName: '',
+    lastName: '',
+    mail: '',
+    rating: '',
+    feedback: ''
+  });
+
+  const radioOptions = [
+    { val: "super-bad", emoji: '\u{1F4A9}' },
+    { val: "bad", emoji: '\u{1F922}' },
+    { val: "mediocre", emoji: '\u{1F623}' },
+    { val: "OK", emoji: '\u{1F914}' },
+    { val: "good", emoji: '\u{1F60A}' },
+    { val: "very-good", emoji: '\u{1F60D}' },
+    { val: "excellent", emoji: '\u{1F37A}' },
+  ]
```

Comment:
please move variable outside of the component body to prevent its re initiating every time the component re-renders